lib/testing.nix

```diff
@@ -39,14 +39,14 @@ rec {
                 , checkPhase # bash (string) actually executing the test
                 , hardware # on what server group should we run this?
                 , needsNixTestEnv ? false # if true, copies over our test env
-                , testNixEnv ? (mkNixTestEnv {}) # qemu images and kernel
+                , testNixEnv ? (mkTestEnv {}) # qemu images and kernel
                 , alwaysSucceed ? false # if true, the build will succeed even on failure and provide a log
                 , ...
                 }@attrs:
     pkgs.stdenv.mkDerivation ((getPCIVars hardware) // {
       src = snabb.src;
 
-      buildInputs = [ pkgs.git pkgs.telnet pkgs.tmux pkgs.numactl pkgs.bc pkgs.iproute pkgs.which pkgs.qemu pkgs.utillinux ];
+      buildInputs = [ pkgs.git pkgs.telnet pkgs.tmux pkgs.numactl pkgs.bc pkgs.iproute pkgs.which pkgs.qemu pkgs.utillinux pkgs.procps ];
```

Comment:
This line could be shorter and simpler:

```
    buildInputs = with pkgs; [ git telnet tmux ... ]
```

Using `with` on a list where everything comes from `pkgs` seems tolerable. :-)